src/Model/List/SS_List.php

```diff
@@ -15,83 +16,243 @@
  */
 interface SS_List extends ArrayAccess, Countable, IteratorAggregate
 {
-
     /**
      * Returns all the items in the list in an array.
      *
      * @return array<T>
      */
-    public function toArray();
+    public function toArray(): array;
 
     /**
      * Returns the contents of the list as an array of maps.
-     *
-     * @return array
      */
-    public function toNestedArray();
+    public function toNestedArray(): array;
 
     /**
      * Adds an item to the list, making no guarantees about where it will
      * appear.
-     *
-     * @param mixed $item
      */
-    public function add($item);
+    public function add(mixed $item): void;
 
     /**
      * Removes an item from the list.
-     *
-     * @param mixed $item
      */
-    public function remove($item);
+    public function remove(mixed $item);
+
+    /**
+     * Return the number of items in this list
+     */
+    public function count(): int;
```

Comment:
```suggestion
```
This is part of the `Countable` interface. We don't need to double require it.

Reply:
Removed